.gitmodules

```diff
@@ -2,3 +2,7 @@
 	path = external/unbound
 	url = https://github.com/monero-project/unbound
 	branch = monero
+[submodule "external/miniupnp"]
+	path = external/miniupnp
+	url = https://github.com/anonimal/miniupnp
```

Comment:
Can you PR this [here](https://github.com/monero-project/miniupnpc)?

Reply:
Yes, as soon as https://github.com/monero-project/miniupnpc/issues/1 is resolved.